modules/src/__init__.py

```diff
@@ -1,5 +1,6 @@
 __all__ = [
     'movie',
     'joke',
-    'hello'
+    'hello',
+    'dictionary'
```

Comment:
Since this list is processed in order, I'd like the `dictionary` module to be just after the `movie` one.